sleuth_backend/solr/models.py

```diff
@@ -69,8 +69,26 @@ class CourseItem(SolrDocument):
         "updatedAt": "",
         "description": "",
         "subjectId": "",
-        "subjectData": []
+        "subjectData": [] # contains subject code [0] and name [1]
     }
 
     def __init__(self, **kwargs):
         super(CourseItem, self).__init__(self.doc, **kwargs)
+
+class RedditPost(SolrDocument):
+    '''
+    Represents a Reddit post
+    '''
+    doc = {
+        "id": "",
+        "type": "redditPost",
+        "name": "",
+        "updatedAt": "",
+        "description": "",
```

Comment:
Since this is the content of the post we it might be better to call this field `content`.

Reply:
`description` is a default field that I've decreed all item types must have